### DISCUSSION TITLE
Month view on iPhone landscape won't show event dots

### DISCUSSION THREAD

**Discussion by phuanggh** [kvyatkovskys/KVKCalendar#330] [General]
Hi Kviatkovskii,
I noticed that event dots on month view is hidden on purpose on iphone landscape (MonthCell.swift: 85), but I can't tell what's the reason behind this decision.
I'd like them to show on Month View. Can I simply remove this line of code? Or should I create an interface for users to configure this themselves?

**Comment by kvyatkovskys** [comment]
Hello @phuanggh! 

I decided disabled the ability to display dots in `MonthView` on phones cos we don't have enough space for it. BTW, you can override cells in `MonthView` to display your custom cell.
